### PR TITLE
Redo entity scale handles to fix scaling into avatars causing undesirable physics/dimensions

### PR DESCRIPTION
**Pull request by dback2**
Fixes bug: https://highfidelity.manuscript.com/f/cases/12943

Test Plan:
https://highfidelity.testrail.net/index.php?/cases/view/11450

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7753/app_12569_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7753/HighFidelity-Beta-PR12569-ad4fc0a60372a657136c733afb117edafba52367.dmg

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7755/app_12569_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7755/HighFidelity-Beta-PR12569-ad4fc0a60372a657136c733afb117edafba52367.dmg

File: scripts/system/libraries/entitySelectionTool.js [hunk 1]
```diff
@@ -1958,7 +1955,11 @@ SelectionDisplay = (function() {
         };
 
         var onMove = function(event) {
-            var proportional = (spaceMode === SPACE_WORLD) || event.isShifted || directionEnum === STRETCH_DIRECTION.ALL;
+			if (event.x === undefined || event.y === undefined) {
```
Comment (misslivirose) [inline scripts/system/libraries/entitySelectionTool.js]: This block needs to be indented an additional space

File: scripts/system/libraries/entitySelectionTool.js [hunk 2]
```diff
@@ -2086,31 +2081,38 @@ SelectionDisplay = (function() {
     function addHandleScaleTool(overlay, mode, directionEnum) {
         var directionVector, offset, selectedHandle;
         if (directionEnum === SCALE_DIRECTION.LBN) {
-            directionVector = { x:1, y:1, z:1 };
+            directionVector = { x:1, y:0, z:1 };
```
Comment (misslivirose) [inline scripts/system/libraries/entitySelectionTool.js]: formatting in this block

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7778/app_12569_PR.apk

**Comment by conklin94122** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7822/app_12569_PR.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7822/HighFidelity-Beta-PR12569-f946c21c4628102328e64c8faa855aac7b834049.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7822/HighFidelity-Beta-PR12569-f946c21c4628102328e64c8faa855aac7b834049.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7869/HighFidelity-Beta-PR12569-e8030d16a3914089bd9feb948c90fa5ae56ce63f.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7875/app_12569_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7875/HighFidelity-Beta-PR12569-190025e02785a0c8fe1f0b429eb009314c7134ce.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7875/HighFidelity-Beta-PR12569-190025e02785a0c8fe1f0b429eb009314c7134ce.exe

**Comment by qlimnavarro** [comment]
Currently Testing

**Comment by qlimnavarro** [comment]
Pass @conklin94122

**Comment by conklin94122** [comment]
@dback2 Needs dev review

File: scripts/system/libraries/entitySelectionTool.js [hunk 3]
```diff
@@ -1999,10 +2017,10 @@ SelectionDisplay = (function() {
             vector = grid.snapToSpacing(vector);
     
             var changeInDimensions = Vec3.multiply(NEGATE_VECTOR, vec3Mult(localSigns, vector));
-            if (directionEnum === STRETCH_DIRECTION.ALL) {
-                var toCameraDistance = getDistanceToCamera(position);
-                var dimensionsMultiple = toCameraDistance * STRETCH_DIRECTION_ALL_CAMERA_DISTANCE_MULTIPLE;
-                changeInDimensions = Vec3.multiply(changeInDimensions, dimensionsMultiple);
+            if (directionEnum === STRETCH_DIRECTION.ALL) {
```
Comment (misslivirose) [inline scripts/system/libraries/entitySelectionTool.js]: minor styling - remove extra spaces at the end of these lines

Comment (dback2) [reply scripts/system/libraries/entitySelectionTool.js]: Will it be a problem that this will differ from the merge back from RC65?

**Pull request closed without merging**